documentation/library_guide/parallel_api/range_based_api.rst

```diff
@@ -1,15 +1,107 @@
 Range-Based API Algorithms
 ##########################
-.. Note::
-
-  The use of the range-based API requires C++17 and the C++ standard libraries coming with GCC 8.1 (or higher)
-  or Clang 7 (or higher).
 
 C++20 introduces the Ranges library. C++20 standard splits ranges into two categories: factories and adaptors.
 A range factory does not have underlying data. An element is generated on success by an index or by dereferencing an iterator.
 A range adaptor, from the |onedpl_long| (|onedpl_short|) perspective, is a utility that transforms the base range,
 or another adapted range, into a view with custom behavior.
 
+|onedpl_short| only supports random access ranges, because they allow efficient and constant-time access to elements at any position in the range. This enables effective workload distribution among multiple threads or processing units, which is crucial for achieving high performance in parallel execution.
+
+|onedpl_short| introduces two sets of range based algorithms:
+
+* The oneapi::dpl::ext::ranges namespace supports integration with the Ranges Library coming with C++20 standard and introduced by std::ranges namespace, allowing you to leverage oneDPL parallel algorithms with the standard ranges paradigm. The functionality is implemented for the host and the device execution policies and requires C++20.
```

Comment:
I am not sure where it should go. Probably the known limitations is a more suitable place. 

There is an ongoing discussion about whether it should be allowed passing ponters-to-members between host and device borders: https://github.com/KhronosGroup/SYCL-Docs/issues/612. It means that it is currently unclear whether it is allowed to pass a pointer-to-member projection into the algorithm with the device policy, and that we should make the users aware.

Reply:
Yes, it should best go to the known limitations. I will add it to another patch I am working on.